source/gameengine/GameLogic/SCA_IObject.cpp

```diff
@@ -248,7 +248,7 @@ SCA_IActuator* SCA_IObject::FindActuator(const std::string& actuatorname)
 }
 
 
-void SCA_IObject::Suspend()
+void SCA_IObject::SuspendLogic()
```

Comment:
@youle31 : This function was used in the original UpdateObjectActivity to disable object logic update, i prefered to use instead of the layer shift.